doc/source/developing/testing.rst

```diff
@@ -346,6 +348,51 @@ test is more useful if you are finding yourself writing a ton of boilerplate
 code to get your image comparison test working.  The ``generic_image`` function is
 more useful if you only need to do a one-off image comparison test.
 
+Updating Image Baselines for pytest-mpl Tests
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+There are 3 situations where updating baseline images may be necessary
+
+- bugfixes
+- intentional change of style in yt
+- old baseline fails with a new version of matplotlib, but changes are not noticeable to the human eye
```

Comment:
I'm blanking this morning and can't remember... do you also do all this when adding a new answer test? or is this only when modifying existing answer tests?

Reply:
The process is intentionally very similar indeed.

Reply:
Oh -- my initial question was vague. What I meant to ask was whether you need to submit the PR to the answer repo when adding a **new** image test (since this section is about "updating" images). And the answer is obviously yes now that I've done that :) I assume that'll be mentioned up above in the TODO section? Or maybe this section should change from "Updating Image Baselines" to "Updating or Adding New Image Baselines"?

Reply:
yes this is what I intended to write where the `TODO` placeholder is, but I think I could never decide what to do about the `generic_image` documentation that we have now, because I never used it and I generally would like to stay away from answer tests as I've had bad experiences trying to debug some.
Anyway, it's a shame that I could never finish this PR as it's clearly needed. If you'd like to take over, be my guest. At a minimum I would need some feedback about my `TODO` (@matthewturk ?).